Dynatrace/dynatrace-operator
# Add feature flag for immutable oneagent images

# Description

This PR adds a feature flag `feature.dynatrace.com/oneagent-immutable-image` which adds connectivity information to the OneAgent pods if enabled.

## How can this be tested?

Deploy cloudNativeFullStack with feature flag enabled and immutable image set in the OneAgent section. 
OneAgent pods should start up and be visible in the UI.


## Checklist
